PublicSDKAPITester/main.m

```diff
@@ -0,0 +1,17 @@
+//
+//  main.m
+//  MigrateTester
+//
+//  Created by Joshua Liebowitz on 6/18/21.
+//
+
+#import <Foundation/Foundation.h>
+@import StoreKit;
+
+int main(int argc, const char * argv[]) {
+    @autoreleasepool {
+        // insert code here...
+        NSLog(@"Hello, World!");
```

Comment:
perhaps we could either remove or replace the comment here with something that explains that we just have this because we _have_ to?

Reply:
💯